### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by mgifford** [GSA/DSSAPIDocumentation#23]
There are a few accessibility issues with this site.

Lots of contrast issues. WAVE & Accessibility Insights both report:

`Element has insufficient color contrast of 4.44 (foreground color: #75787b, background color: #ffffff, font size: 16.5pt (22px), font weight: normal). Expected contrast ratio of 4.5:1`

`Element has insufficient color contrast of 4.35 (foreground color: #0072ce, background color: #f1f2f2, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1`

and with language:

`The <html> element does not have a lang attribute`
